packages/eslint-plugin-pf-codemods/src/rules/v6/formFiledGroupHeaderTitleTextObjectRenamed/formFiledGroupHeaderTitleTextObject-renamed.ts

```diff
@@ -41,16 +41,18 @@ module.exports = {
           callContextReport(node, node.imported);
         }
       },
+      ExportSpecifier(node: ExportSpecifier) {
+        if (!importHasAlias && node.local.name === previousName) {
+          callContextReport(node, node.local);
+        }
```

Comment:
Couple more things we should do for this:

- The export shouldn't be reliant on the import not being aliased, so we should remove the !importHasAlias` portion of this conditional.
- We only want to make this update if the export is from Patternfly, so we'd need to update line 9 to destructure both the imports and exports, then we want to only make an update in this ExportSpecifier block if an `interfaceExport` exists.

Reply:
I misunderstood how the exporting can work. I thought it is necessary to first import the name to be able to reexport it, so the codemod only covers a case like this:

```
import { FormFiledGroupHeaderTitleTextObject } from "@patternfly/react-core";
export { FormFiledGroupHeaderTitleTextObject as HeaderTitleObject };
```

If I understand your review correctly, we want to cover this:
```
export { FormFiledGroupHeaderTitleTextObject as HeaderTitleObject } from "@patternfly/react-core";
```

But shouldn't we still want to have the case above covered by the codemod?